test/Kestrel.FunctionalTests/AddressRegistrationTests.cs

```diff
@@ -169,9 +169,16 @@ public class AddressRegistrationTests : TestApplicationErrorLoggerLoggedTest
                 {
                     var response = await HttpClientSlim.GetStringAsync(testUrl, validateCertificate: false);
 
-                    // Compare the response with Uri.ToString(), rather than testUrl directly.
-                    // Required to handle IPv6 addresses with zone index, like "fe80::3%1"
-                    Assert.Equal(new Uri(testUrl).ToString(), response);
+                    // Filter out the scope id for IPv6, that's not sent over the wire. "fe80::3%1"
```

Comment:
Add `https://github.com/aspnet/Common/pull/369` to this comment so it's obvious why we're filtering this out.